packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-file-upload/question.js

```diff
@@ -106,35 +107,36 @@ class MultiFileUploadQuestion extends Question {
 			// save answer to database
 			await patchQuestionResponse(journeyResponse.journeyId, encodedReferenceId, responseToSave);
 
-			// todo: validation after saving required?
-			// if (Object.keys(errors).length > 0) {
-			// 	const answer = journeyResponse.answers[this.fieldName].uploadedFiles;
-			// 	return this.renderPage(
-			// 		res,
-			// 		{
-			// 			layoutTemplate: journey.journeyTemplate,
-			// 			pageCaption: sectionObj?.name,
-			// 			backLink: journey.getNextQuestionUrl(sectionObj.segment, this.fieldName, true),
-			// 			listLink: journey.baseUrl,
-			// 			answers: journey.response.answers,
-			// 			answer
-			// 		},
-			// 		{
-			// 			errors,
-			// 			errorSummary: errorSummary.map((error) => ({
-			// 				...error,
-			// 				href: '#'
-			// 			}))
-			// 		}
-			// 	);
-			// }
-
 			// map the response back to journeyResponse
 			journeyResponse.answers[this.fieldName] = responseToSave.answers[this.fieldName];
 
 			// update journey based on response
 			const updatedJourney = new journey.constructor(journeyResponse);
 
+			// todo: validation after saving required?
```

Comment:
not sure what this means?

Reply:
Can remove, I wasn't sure with how the validation was going to work in the dynamic forms, and if we would be expecting to continue to show validation messages after having saved files to blob storage or not.